backend/action_logs/serializers.py

```
@@ -123,4 +133,43 @@ def to_representation(self, instance):
+        try:
+            data = super().to_representation(instance)
+            # Ensure all required fields are present
+            if 'user' in data and not data['user'].get('email'):
+                data['user']['email'] = ''
+            if 'status' not in data:
+                data['status'] = 'open'
+            if 'is_approved' not in data:
+                data['is_approved'] = False
+            if 'is_viewed' not in data:
+                data['is_viewed'] = False
+            return data
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)